view/src/service.rs

```diff
@@ -233,4 +234,21 @@ impl ViewProtocol for ViewService {
         };
         Ok(tonic::Response::new(witness_data.into()))
     }
+
+    async fn chain_params(
+        &self,
+        _request: tonic::Request<pb::ChainParamsRequest>,
+    ) -> Result<tonic::Response<pbp::ChainParams>, tonic::Status> {
+        self.check_worker().await?;
+
+        // NOTE: should we check the chain ID here?
```

Comment:
Yeah, probably. There's a merge conflict now, so I'll rebase the PR and add the check while I do.

Reply:
Hmm, wait. How does the wallet client know the chain ID? It's supposed to be stateless.

Reply:
Removed the parameter in https://github.com/penumbra-zone/penumbra/pull/893/commits/be562908215a29135414917b19b51831a446b390